core/services/fluxmonitorv2/orm_test.go

```diff
@@ -0,0 +1,251 @@
+package fluxmonitorv2_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/core/assets"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/services/fluxmonitorv2"
+	"github.com/smartcontractkit/chainlink/core/services/job"
+	"github.com/smartcontractkit/chainlink/core/services/pipeline"
+	"github.com/smartcontractkit/chainlink/core/services/postgres"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/stretchr/testify/require"
+)
+
+func TestORM_FindOrCreateFluxMonitorRoundStats(t *testing.T) {
```

Comment:
I think these first three tests could just be part of a single test where we create, count/read then delete. Always want to minimize the number of NewStores to keep the db load low when running the suite

Reply:
Done